### PR TITLE
Use interactive components from Quo

### PR DESCRIPTION
**Pull request by Ferossgp**
This PR introduces Quo components to the Status App. All components are introduced in one PR to enable animations to all components at once because the Gesture Handler was changed. Code that was not using components was refactored, thus UI regressions may be introduced. 

Components updated and refactored:
1. Toolbar
2. Buttons
3. Icon Buttons
4. Bottom sheet

### CODE REVIEW

File: .env.release [hunk 1]
```diff
@@ -18,3 +18,4 @@ RPC_NETWORKS_ONLY=1
 PARTITIONED_TOPIC=0
 MOBILE_UI_FOR_DESKTOP=1
 HARDWALLET_ENABLED=1
+DISABLE_ROOT_ALERT=0
```
Comment (flexsurfer) [inline .env.release]: probably better to name it ENABLE_ROOT_ALERT in that case it won't be enabled in release by mistake

File: src/status_im/ui/screens/browser/accounts.cljs [hunk 2]
```diff
@@ -2,16 +2,16 @@
...
             [status-im.utils.utils :as utils]
             [re-frame.core :as re-frame]))
 
 (defn render-account [dapps-account]
   (fn [account]
-    [list-item/list-item
-     {:theme     :selectable
-      :selected? (= (:address dapps-account) (:address account))
+    [quo/list-item
+     {:accessory :radio
```
Comment (flexsurfer) [inline src/status_im/ui/screens/browser/accounts.cljs]: whole item should be touchable

**Review by flexsurfer — APPROVED** [review]

**Comment by errorists** [comment]
it's glorious but I'm unsure about haptic feedback on Android, on the Samsung Galaxy phone I tried it's absolutely godawful, like if being jolted with electricity when touching the screen. If that's the standard there's no way we should leave that enabled on Android.

**Comment by oskarth** [comment]
Small request - could you please rebase to latest master so this commit https://github.com/status-im/status-react/commit/b5fda1234e5447cbc626059dfc78fcfbabc9e926 gets included? This means we aren't running E2E tests on eth.prod cluster anymore, and since this happens on every push, the impact is quite big. This ensures we have more accurate metrics going forward from Jul 1 onward, which would be awesome. See https://discuss.status.im/t/user-growth-and-retention/1782 for more

**Comment by Serhy** [comment]
@Ferossgp thanks a lot for huge work done here and for fixing regressions
Looks good to me now for merge based on tests i've made on Android 8/9 and iOS13.

**Pull request merged**